src/controllers/sqlOutput.ejs

```diff
@@ -1,36 +1,17 @@
 <html>
   <head>
-    <title><%=docTitle%></title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <style>
-      :root {
-        --background-color: <%=backgroundcolor%>;
-        --color: <%=color%>;
-        --font-family: <%=fontfamily%>;
-        --font-weight: <%=fontweight%>;
-        --font-size: <%=fontsize%>;
-      }
-      body {
-        background-color: var(--background-color);
-        color: var(--color);
-        font-family: var(--font-family);
-        font-weight: var(--font-weight);
-        font-size: var(--font-size);
-      }
-    </style>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' vscode-resource: https:; script-src vscode-resource: 'self' 'unsafe-inline' 'unsafe-eval' https:; style-src vscode-resource: 'self' 'unsafe-inline';"/>
+    <base href="<%=basehref%>">
     <link rel="stylesheet" type="text/css" href="lib/css/slick.grid.css"/>
     <link rel="stylesheet" type="text/css" href="lib/css/SlickGrid.css"/>
     <link rel="stylesheet" type="text/css" href="dist/css/styles.min.css"/>
-    <%if (theme === 'vscode-light') { %>
```

Comment:
I think we are still going to want this, I just didn't implement it when I was doing this. You should be able to get the theme style from vscode's apis

Reply:
I added the tags to the css file instead to handle the specific styling. Will find a better way to imitate the conditional styling.